ASSETS.md

```diff
@@ -39,6 +39,26 @@ requirement:
     version.min: 7.1
   filebeat:
     version.min: 7.2
+
+# The order of the items listed here is the order they show up in the package overview.
+screenshots:
+  # The src path is relative from inside the package. The full path will be generated by the server
+  # and exposed through the API.
+- src: /img/overview-logs.png
+  title: This shows the overview of the logs dashboard.
+  # The type does not have to be set explicitly if it's in the file extesions
+  # but the server will extract it an expose it through the API
+  type: image/png
+  # The size of the image could be detect by the server too if needed.
+  # We must come up with a recommended image size.
+  size: 800x600
+- src: /img/overview-metrics.jpg
+  title: Metrics Dashboard.
+  size: 800x600
+- src: getting-started.mp4
+  title: Getting started with the envoyproxy integration.
+  size: 800x600
```

Comment:
@jfsiii For icons we discuss to make this an array. I would suggest for screenshots we don't for now.